extensions/src/tools/component-wrapper.js

```diff
@@ -47,6 +48,16 @@ function createExtension(component) {
       this.component.$el.remove();
       this.component = null;
     }
+
+    /**
+     *
+     * Set the authorization so all the extensions can interact with nexus.
+     *
+     * @param {string} token - Bearer token (with Bearer string included)
+     */
+    static setAuthToken(token) {
```

Comment:
With this setup setAuthToken will be defined on a local extension level, but will mutate a configuration of axios from upper scope. To be consistent it's better to move it to extensions module (./src/components/extensions/index.js).

So one use it like:
```
import extensions from '@bbp/nexus-search-webapp-extensions'

extensions.setAuthToken(token);
```